group_vars/clients

```diff
@@ -1,9 +1,8 @@
 # Local directory containing HTTP replay files for NDT E2E client worker.
 local_http_replay_dir: http_replays
 
-# HTTP replay file for Banjo client (note that this file is not in this git
-# repo and must be generated or copied by an alternative means).
-http_replay_file: replay-banjo-2016-06-07.yaml
+# HTTP replay file for OneBox client
+http_replay_file: onebox-2016-06-07.yaml
```

Comment:
Nope.  It's using the same old file.  I just gave it more intuitive name, and left the date part so that we'd know when it was generated. I had a somewhat detailed email conversation with Michael, and it seems that we should stick with this older replay until we can get access to a sandboxed version of the OneBox site again.